lang/ilias_ka.lang

```diff
@@ -1343,7 +1343,7 @@ chatroom_adm#:#client_settings#:#ზოგადი პარამეტრე
 chatroom_adm#:#enable_osd#:#ფანჯრის შეტყობინებების გააქტიურება
 chatroom_adm#:#general_settings_title#:#General Chat Settings
 chatroom_adm#:#https#:#HTTPS
-chatroom_adm#:#osd_intervall#:#ინტერვალის განახლება (წამები)
+chatroom_adm#:#osd_interval#:#ინტერვალის განახლება (წამები)
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.